ush/forecast_postdet.sh

```diff
@@ -217,6 +220,21 @@ EOF
         cpreq "${increment_file}" "${DATA}/INPUT/${inc_file}"
       done
 
+      # Land IAU increments: sfc_inc in FV3 grid, all timesteps in one file per tile 
+      if [[ ${DO_LAND_IAU} = ".true." ]]; then
+        local TN sfc_increment_file
+        for TN in $(seq 1 "${ntiles}"); do
+          sfc_increment_file="${COMIN_ATMOS_ANALYSIS}/sfc_inc.tile${TN}.nc"
+          if [[ ! -f "${sfc_increment_file}" ]]; then
+            echo "FATAL ERROR: DO_LAND_IAU=${DO_LAND_IAU}, but missing increment file ${sfc_increment_file}, ABORT!"
+            exit 1
+          fi
+          cpreq "${sfc_increment_file}" "${DATA}/INPUT/sfc_inc.tile${TN}.nc"
+        done
+        LAND_IAU_INC_FILES=${LAND_IAU_INC_FILES:-'sfc_inc'}
```

Comment:
LAND_IAU_INC_FILES is hard coded above to be sfc_inc, so there's no need in the workflow to use a variable here.